lib/core/device.rb

```diff
@@ -1364,6 +1364,31 @@ def wait_not_visible(action)
             "#{default_wait_time} seconds\nError Screenshot: #{path}"
   end
 
+  # guarantee that the checkbox is checked or unched depending on the option
+  # Accepts:
+  #   Strategy
+  #   Id
+  #   Option -> check or uncheck
+  def credentials_checkbox(action)
+    action = convert_value_pageobjects(action);
+    option = convert_value(action["Option"])
+    el = @driver.find_element(convert_value(action["Strategy"]), convert_value(action["Id"]))
+    input = el.find_element(:xpath => "./input")
+    span = el.find_element(:xpath => "./span")
+    is_checked = false
+    
+    if input.attribute("checked")
+      is_checked = true
+    end
+      
+    if (is_checked) && (option == "uncheck")
+      span.click
+    elsif (option == "check") && (!is_checked)
+      span.click
+    else
```

Comment:
it seems that there is an empty else on this part.

Reply:
resolved